### ISSUE TITLE
Reparameterization

### ISSUE THREAD

**Issue by zixinixiz** [fishmoon1234/DAG-GNN#8]
Hi,

I am confused about the construction of the encoder and decoder. It seems that there is no reparameterization step in your code. In other words, you directly used M_x/M_z as x/z. Is there any problem?

Looking forward to your reply.

**Comment by fishmoon1234** [comment]
Yes you are right. We are using M_x as x and M_z as z by default.
